metabrainz/templates/payments/payment.html

```diff
@@ -189,7 +189,7 @@ <h1 class="page-title">Make a Payment</h1>
       var stripeHandler = StripeCheckout.configure({
         key: "{{ stripe_public_key }}",
         name: "MetaBrainz Foundation",
-        description: "Payment to the MetaBrainz Foundation",
+        description: "{{ _('Payment to the ') }}MetaBrainz Foundation",
```

Comment:
According to me, there's only `donate.html` (https://github.com/metabrainz/metabrainz.org/pull/248/files#diff-91c562dad3573e5c56c4a8a143d7e0a0) that handles payments except this, where such changes were missed.